### ISSUE TITLE
Query explorer fails to load the page and GA account linkage

### ISSUE THREAD

**Issue by The-Edgar** [googleanalytics/ga-dev-tools#262]
I get a grey screen and an infinite loading bar when I try to open [(https://ga-dev-tools.appspot.com/query-explorer/)] in Chrome v 55.0.2883.95 (64-bit). There is no problem when opening the page and linking my account in Firefox browser.

The previous issue tickets led me to look at the JS Console which gives the following image:
<img width="543" alt="screen shot 2017-01-02 at 3 04 55 pm" src="https://cloud.githubusercontent.com/assets/11462599/21589692/25ac25c0-d0fd-11e6-9f6a-9d13ecfe0919.png">

I get the same errors when I open the page in incognito where all my extensions and addons are disabled.

However, I do not know anything about Javascript and I dont know what to do with these errors. Clicking on the first link in the first three error messages points to this line in the source code:
"options = SAFARI ? safari.extension.settings : localStorage;"

The last error message points to something very messy related to sessionStorage.

Any ideas what to do?

**Comment by philipwalton** [comment]
It looks to me like you have an extension or setting that's blocking access to local storage. I know you said you've tried in an incognito window with all extensions and add-ons disabled, but all these error messages are saying access to local storage denied, which is not normal, and seems like something you've opt-ed into somehow.

**Comment by jjmalina** [comment]
FWIW I had a similar problem. This was in Chrome Incognito with no extensions enabled.
![screenshot from 2017-03-21 15 40 48](https://cloud.githubusercontent.com/assets/178138/24167366/dcc608fc-0e4c-11e7-824e-e0d3245fd3c5.png)

Then I discovered that I had the "Block third-party cookies and site data" setting checked off

**Comment by Lucretiel** [comment]
@The-Edgar can you confirm jjmalina's fix? Ensure that this isn't being caused by an extension or a privacy setting?

**Comment by Lucretiel** [comment]
@jjmalina Third party cookies are required for the site to work, but the fact that they make the UI hang is a known issue: #284.

**Comment by Lucretiel** [comment]
I'm presuming this is fixed (per the thread) and closing the issue. Feel free to reopen if you have more issues.

**Comment by Gamlilorien** [comment]
Hmm I seem to be having a similar problem in that when I visit: https://ga-dev-tools.appspot.com/account-explorer and then authenticate with my account, I then get a never ending loading screen.
<img width="1257" alt="Screen Shot 2019-06-28 at 2 30 04 PM" src="https://user-images.githubusercontent.com/42821488/60369740-e0208700-99b1-11e9-88db-457f2d175616.png">
I have verified my settings for Cookies (as suggested), and I have tried different browsers, all to no avail.
<img width="671" alt="Screen Shot 2019-06-28 at 2 36 12 PM" src="https://user-images.githubusercontent.com/42821488/60369962-79e83400-99b2-11e9-9831-d96c6e34390b.png">

Is there something else that I am missing? Any help or suggestions would be greatly appreciated!
My Browser and OS Specs:
Chrome: Version 75.0.3770.100 (Official Build) (64-bit)
Mac OSX: 10.14.5
